src/ConcreteOperations/intersection.jl

```diff
@@ -710,11 +713,16 @@ clipping algorithm](https://en.wikipedia.org/wiki/Sutherland%E2%80%93Hodgman_alg
 The implementation is based on the one found in
 [rosetta code](http://www.rosettacode.org/wiki/Sutherland-Hodgman_polygon_clipping#Julia).
 """
-function intersection(P1::VPolygon, P2::VPolygon)
+function intersection(P1::VPolygon{N}, P2::VPolygon{N};
+                      apply_convex_hull::Bool=true) where {N}
     v1 = vertices_list(P1)
     v2 = vertices_list(P2)
     v12 = _intersection_vrep(v1, v2)
-    return VPolygon(v12)
+    if isempty(v12)
+        return EmptySet{N}(2)
+    else
+        return VPolygon(v12, apply_convex_hull=apply_convex_hull)
```

Comment:
IMO `VPolygon`s should always be created using a convex hull unless the algorithm guarantees that result anyway.

Reply:
yes , i agree. i left the option as `true` by default.